sample/src/main/java/com/decathlon/compose/sample/screens/Chips.kt

```diff
@@ -0,0 +1,513 @@
+            VitaminChips.Action(
+                label = "Re fill",
+                enabled = inputChipsEnabled,
+                onClick = {
+                    displayedContacts.clear()
+                    displayedContacts.addAll(listOf("John", "Alice", "Helen", "Charles"))
+                },
+                startIcon = {
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_vtmn_refresh_line),
+                        contentDescription = null
+                    )
+                }
+            )
+
+            displayedContacts.forEach {
+                VitaminChips.Input(
+                    label = it,
+                    selected = false,
+                    enabled = inputChipsEnabled,
+                    onClick = {
+                        //nothing
+                    },
+                    removeContent = {
+                        Close(
+                            onClick = { displayedContacts.remove(it) },
+                            contentDescription = "Remove"
+                        )
+                    }
+                )
+            }
+            Spacer(modifier = Modifier.width(16.dp))
+        }
+    }
+}
+
+@ExperimentalMaterialApi
+@Composable
+private fun SingleChoiceChips(singleChoiceEnabled: Boolean) {
+    Row(
+        modifier = Modifier
+            .padding(top = 8.dp, bottom = 8.dp)
+            .horizontalScroll(rememberScrollState()),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.spacedBy(8.dp)
+    ) {
+        Spacer(modifier = Modifier.width(16.dp))
+
+        var selected by remember { mutableStateOf("Medium") }
+        listOf("Easy", "Medium", "Hard", "Expert").forEach { value ->
+            VitaminChips.Input(
+                label = value,
+                enabled = singleChoiceEnabled,
+                selected = value == selected,
+                onClick = {
+                    selected = value
+                }
+            )
+        }
+
+        Spacer(modifier = Modifier.width(16.dp))
+    }
+    Row(
+        modifier = Modifier
+            .padding(top = 8.dp, bottom = 8.dp)
+            .horizontalScroll(rememberScrollState()),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.spacedBy(8.dp)
+    ) {
+        Spacer(modifier = Modifier.width(16.dp))
+
+        var smallSelected by remember { mutableStateOf("Hard") }
+        listOf("Easy", "Medium", "Hard", "Expert").forEach { value ->
+            VitaminChips.Input(
+                label = value,
+                enabled = singleChoiceEnabled,
+                selected = value == smallSelected,
+                sizes = VitaminChipSizes.small(),
+                onClick = {
+                    smallSelected = value
+                }
+            )
+        }
+
+        Spacer(modifier = Modifier.width(16.dp))
+    }
+}
+
+@ExperimentalFoundationApi
```

Comment:
What is experimental ?

Reply:
It is because I am using Surface component with onClick handling paramethor